### PR TITLE
Cromwell piece of hashing, closes #490

### LINKED ISSUE #490: Move hash to wdl4s + backend

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: src/main/scala/cromwell/engine/WorkflowDescriptor.scala [hunk 1]
```diff
@@ -171,6 +171,10 @@ case class WorkflowDescriptor(id: WorkflowId,
   private def logReadDisabled() = workflowLogger.warn(readDisabled)
 
   override def toString = s"WorkflowDescriptor(${id.id.toString})"
+
+  def getHash(wdlValue: WdlValue): Option[SymbolHash] = {
```
Comment (geoffjentry) [inline src/main/scala/cromwell/engine/WorkflowDescriptor.scala]: shouldn't this just be `hash`?

Comment (mcovarr) [inline src/main/scala/cromwell/engine/WorkflowDescriptor.scala]: Yes.  This is how I found it, but I'll change it.

File: src/main/scala/cromwell/engine/SymbolStoreEntry.scala [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package cromwell.engine
 
 import wdl4s.types.WdlType
-import wdl4s.values.WdlValue
+import wdl4s.values.{SymbolHash, WdlValue}
```
Comment (cjllanwarne) [inline src/main/scala/cromwell/engine/SymbolStoreEntry.scala]: I don't see why this would be necessary since nothing else changed?

Comment (mcovarr) [inline src/main/scala/cromwell/engine/SymbolStoreEntry.scala]: `SymbolHash` now lives in wdl4s, it used to live in the engine package object.

File: src/main/scala/cromwell/util/FileUtil.scala [hunk 3]
```diff
@@ -4,8 +4,8 @@ import java.io.{File, FileInputStream, Writer}
 import java.nio.file.Path
 
 import better.files._
-import cromwell.engine.Hashable
 import org.apache.commons.codec.digest.DigestUtils
+import wdl4s.values.Hashable
```
Comment (cjllanwarne) [inline src/main/scala/cromwell/util/FileUtil.scala]: same here?

**Comment by cjllanwarne** [comment]
`sbt.ResolveException: unresolved dependency: org.broadinstitute#wdl4s_2.11;0.4: not found` ?

**Comment by cjllanwarne** [comment]
:+1:

**Comment by mcovarr** [comment]
re the build breakage, I need to have the wdl4s snapshot published by the other PR's merge to develop and then update the build.sbt to reference that.

File: build.sbt [hunk 4]
```diff
@@ -11,7 +11,7 @@ version := "0.19"
 
 val lenthallV = "0.16"
 
-val wdl4sV = "0.3"
+val wdl4sV = "0.4"
```
Comment (mcovarr) [inline build.sbt]: actually I think this should be 0.3 at a snapshot hash

Comment (scottfrazer) [inline build.sbt]: 0.4 + snapshot?  We already released 0.3 I believe.

**Comment by scottfrazer** [comment]
:+1:

File: build.sbt (hunk 4 above)
Comment (mcovarr) [inline build.sbt]: uh yeah, that makes a lot more sense :frowning:

**Pull request merged**